tests/readPreference/readpreference-ctor-001.phpt

```diff
@@ -7,8 +7,9 @@ MongoDB\Driver\ReadPreference construction
 require_once __DIR__ . "/../utils/basic.inc";
 
 var_dump(new MongoDB\Driver\ReadPreference(MongoDB\Driver\ReadPreference::RP_PRIMARY));
-var_dump(new MongoDB\Driver\ReadPreference(MongoDB\Driver\ReadPreference::RP_SECONDARY, array(array("tag" => "one"))));
-var_dump(new MongoDB\Driver\ReadPreference(MongoDB\Driver\ReadPreference::RP_PRIMARY, array()));
+var_dump(new MongoDB\Driver\ReadPreference(MongoDB\Driver\ReadPreference::RP_SECONDARY, [['tag' => 'one']]));
+var_dump(new MongoDB\Driver\ReadPreference(MongoDB\Driver\ReadPreference::RP_PRIMARY, []));
```

Comment:
Just `array()` to `[]` refactoring here. Since this is the only ReadPreference constructor test, I added a new case for "maxStalenessMS" below; however, it's probably redundant in light of the `ReadPreference::getMaxStalenessMS()` test.